src/screens/Settings/MerchantAccountUtils.res

```diff
@@ -419,7 +414,17 @@ let validateCustom = (key, errors, value, isLiveMode) => {
         "Please enter integer value from 1 to 5"->JSON.Encode.string,
       )
     }
+  | _ => ()
+  }
+}
 
+let validateEmptyValue = (key, errors) => {
+  switch key {
+  | MaxAutoRetries =>
+    errors->Dict.set(
+      key->validationFieldsMapper,
+      "Please enter a max auto retries value"->JSON.Encode.string,
+    )
```

Comment:
If the field is a numeric input then i guess you can add the condition in the same validate custom func

Reply:
okay, updated